src/components/table/ItemNameCellRenderer.js

```diff
@@ -21,25 +22,28 @@ const ItemNameCellRenderer = (showThumbnails) => {
   const Component = ({ data: item }) => {
     const classes = useStyles();
     const { t } = useTranslation();
-    const ThumbnailComponent = showThumbnails
-      ? Thumbnail({
-          id: item.id,
-          extra: item.extra,
-          maxWidth: 30,
-          maxHeight: 30,
-          alt: t('small thumbnail'),
-          useThumbnail: hooks.useItemThumbnail,
-        })
-      : null;
+
+    const alt = t('small thumbnail');
+    const defaultValueComponent = (
+      <ItemIcon
+        type={item.type}
+        id={item.id}
+        extraIcon={getEmbeddedLinkExtra(item.extra)?.icons?.[0]}
+        name={item.name}
+      />
+    );
 
     return (
       <div className={classes.nameCell} id={buildNameCellRendererId(item.id)}>
-        {ThumbnailComponent ?? (
-          <ItemIcon
-            type={item.type}
+        {showThumbnails && (
+          <Thumbnail
             id={item.id}
-            extra={item.extra}
-            name={item.name}
+            extraThumbnail={getEmbeddedLinkExtra(item.extra)?.thumbnails?.[0]}
+            maxWidth={30}
+            maxHeight={30}
```

Comment:
No more `type`? Why add the size here?

Reply:
<img width="346" alt="Screenshot 2022-08-12 at 11 41 51" src="https://user-images.githubusercontent.com/67112345/184328819-af44e6e0-b742-4d83-b499-c8b7f2c3dd86.png">
This was the Thumbnail component we were passing before the refactor (and that caused the error because it was provided as a function instead of a JSX element). I am passing the same props. I do not know what you mean with the type (we were not passing it, and there is no `type` prop in Thumbnail), and the max width and height was passed before the refactor from here.

Reply:
I am also passing now a defaultValue prop to solve the error of the function component